README.md

```diff
@@ -1,34 +1,21 @@
 # GslibIO.jl
 
-Utilities to read/write *extended* [GSLIB](http://www.gslib.com/gslib_help/format.html) files in Julia.
+Utilities to read/write *extended* and *legacy* [GSLIB](http://www.gslib.com/gslib_help/format.html) files in Julia.
 
 [![Build Status](https://img.shields.io/github/workflow/status/JuliaEarth/GslibIO.jl/CI)](https://github.com/JuliaEarth/GslibIO.jl/actions)
 [![Coverage Status](https://codecov.io/gh/JuliaEarth/GslibIO.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaEarth/GslibIO.jl)
 
-## Introduction
+## Overview
 
-The GSLIB file format was introduced a long time ago for storing regular grids in text files that are easy to read. The format specification is incomplete mainly because:
+The GSLIB file format was introduced a long time ago for storing spatial data over regular grids
+and point sets in text files that are easy to read. The format specification is incomplete mainly because:
 
 1. it doesn't contain the size of the grid (i.e. `(nx, ny, nz)`)
 2. it doesn't specify the origin and spacing (i.e. `(ox, oy, oz)`, `(sx, sy, sz)`)
 3. it doesn't specify the special symbol for inactive cells (e.g. `-999`)
+4. it doesn't specify which variable names are coordinates
```

Comment:
I would start the sentences in 1-3 with "for the grid case", because for point sets are not used

Reply:
Nice. I will split 1-3 from 4 tomorrow morning.